Packages/io.chainsafe.web3-unity.walletconnect-chainsafe/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "io.chainsafe.web3-unity.walletconnect-chainsafe",
```

Comment:
wait didn't we say that we're leaving it as part of our core package

Reply:
we've decided we gonna keep the new one in the core and provide the one created by me in a separate package.